testing/grafana-image-renderer/10-host-param.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/src/app.ts b/src/app.ts
```

Comment:
Is this patch upstream?

Reply:
Yes https://github.com/grafana/grafana-image-renderer/pull/40